lib/erc20/Erc20Rpc.js

```diff
@@ -19,6 +19,11 @@ class Erc20RPC extends EthRPC {
     const contractData = this.erc20Contract.methods
       .transfer(address, amountStr)
       .encodeABI();
+
+    if (passphrase) {
+      this.emitter.emit('unlockedForOne');
+    }
```

Comment:
We also need to emit `unlocked` when the transaction has either been sent or there is an error